docs/source/reference/exception.rst

```diff
@@ -0,0 +1,73 @@
+############
+ Exceptions
+############
+
+In TensorBaySDK, some custom exceptions are defined to distiguish from the built-in exceptions of python.
```

Comment:
TensorBay SDK

distinguish

Exceptions are not defined to distinguish from build-in exceptions.